states.go

```diff
@@ -52,6 +52,8 @@ const (
 	// UpdateHubStateWaitingForReboot is set when the agent is waiting
 	// for reboot
 	UpdateHubStateWaitingForReboot
+	// UpdateHubStateFinal is set when the daemon is about to quit
+	UpdateHubStateFinal
```

Comment:
Isn't better to call it Exit? I believe it makes clear that this is going to quit the daemon (or Quit).